onnxruntime/contrib_ops/cpu/transformers/subgraph_t5_decoder.cc

```diff
@@ -156,9 +156,15 @@ Status T5DecoderSubgraph::CreateInitialFeeds(
   // of encoder.
   // When first_past_input_index_ == 2, the past states are copied from the second output of encoder.
   for (size_t j = 4 - first_past_input_index_; j < encoder_fetches.size(); j++) {
-    OrtValue expanded_cache;
-    BeamSearchCpuDeviceHelper::ExpandCaches<float>(encoder_fetches[j], num_beam, allocator, expanded_cache);
-    decoder_feeds.push_back(expanded_cache);
+    if (j == 1) {
```

Comment:
hidden_states is optional, need add a check that there is hidden state.

Reply:
check added